src/services/editors/entity-editor/useEntityEditor.ts

```diff
@@ -1,19 +1,67 @@
-"use client"
-
 import { create } from "zustand"
 import { ClapEntity } from "@aitube/clap"
 import { EntityEditorStore } from "@aitube/clapper-services"
+import { useTimeline } from "@aitube/timeline"
 
 import { getDefaultEntityEditorState } from "./getDefaultEntityEditorState"
 
 export const useEntityEditor = create<EntityEditorStore>((set, get) => ({
   ...getDefaultEntityEditorState(),
-  setCurrent: (current?: ClapEntity) => { set({ current }) },
+  current: undefined,
+  entities: [],
+  setCurrent: (current?: ClapEntity) => set({ current }),
+  selectEntity: (id: string) => {
+    const entity = get().entities.find(e => e.id === id)
+    set({ current: entity })
+  },
   undo: () => {},
   redo: () => {},
-}))
-
-
-if (typeof window !== "undefined") {
-  (window as any).useEntityEditor = useEntityEditor
-}
```

Comment:
I propose to keep this, it is ugly but I use this for debugging (inspect the state of the app at runtime in the chrome developer console)